### ISSUE TITLE
Missing heading mark-up

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds.com#17]
For example, from # 17: Text  such as "Transition to Full Compliance", "Master Client Index", "Option one", "Option two", etc. are not marked up using html heading mark-up.
This resulted in screen reader users finding it difficult to understand the page content structure.

Code Snippet:
> \<p><b>Transition to Full Compliance</b></p>
> (..)
> \<p><i>Option one</i></p>

Including but not limited to - please check for this existing elsewhere:

Page Name | Accessibility Report Issue Number
------------|-----------------------------------
Global : Header & Footer | # 17

Numbers are the issue numbers from the Accessibility Report on the Findings worksheet at https://docs.google.com/spreadsheets/d/1fQgaUcbWK5n8gUorLlAoITfGaJC0Gqg1jzuur9wPoZg/edit#gid=585536028
